examples/hotrod/services/driver/redis.go

```diff
@@ -35,27 +35,32 @@ import (
 
 // Redis is a simulator of remote Redis cache
 type Redis struct {
-	tracer opentracing.Tracer // simulate redis as a separate process
+	tracer trace.Tracer // simulate redis as a separate process
 	logger log.Factory
 	errorSimulator
 }
 
 func newRedis(otelExporter string, metricsFactory metrics.Factory, logger log.Factory) *Redis {
 	return &Redis{
-		tracer: tracing.Init("redis", otelExporter, metricsFactory, logger),
+		tracer: tracing.InitOTEL("redis-manual", otelExporter, metricsFactory, logger),
 		logger: logger,
 	}
 }
 
 // FindDriverIDs finds IDs of drivers who are near the location.
 func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
-	if span := opentracing.SpanFromContext(ctx); span != nil {
+	_, span := r.tracer.Start(ctx, "FindDriverIDs", trace.WithSpanKind(trace.SpanKindClient))
+	span.SetAttributes(attribute.Key("param.driver.location").String(location))
+
+	defer span.End()
+
+	/* if span := opentracing.SpanFromContext(ctx); span != nil {
```

Comment:
Can the commented-out opentracing instrumentation be removed?

Reply:
I think it can be taken as reference to anyone who wants re-instrument their application this way.

Reply:
I agree w/ @albertteoh - it's never a good idea to keep commented out code.